werkzeug/serving.py

```diff
@@ -278,9 +282,42 @@ def log_message(self, format, *args):
         self.log('info', format, *args)
 
     def log(self, type, message, *args):
-        _log(type, '%s - - [%s] %s\n' % (self.address_string(),
-                                         self.log_date_time_string(),
-                                         message % args))
+        msg = '%s - - [%s] %s' % (self.address_string(),
+                                  self.log_date_time_string(),
+                                  message % args)
+
+        # HTTP Status Code
+        code = str(args[1])
+
+        # 1xx - Informational
+        elif code[0] == '1':
+            msg = colored(msg, attrs=['bold'])
+
+        # 2xx - Success
+        if code[0] == '2':
+            msg = colored(msg, color='white')
+
+        # 304 - Resource Not Modified
+        elif code == '304':
+            msg = colored(msg, color='cyan')
+
+        # 3xx - Redirection
+        elif code[0] == '3':
+            msg = colored(msg, color='green')
+
```

Comment:
Too many spaces IMO, please move the comments inline to be after the `:` and remove empty lines inbetween.